tests/quantization/test_quantized_ops.py

```diff
@@ -1146,6 +1148,60 @@ def test_quantized_reshape(shape):
     )
 
 
+@pytest.mark.parametrize(
+    "original_shape, expand_shape",
+    [
```

Comment:
great thorough tests ! But maybe there was a way to automate this ? How can we be sure that all interesting cases are covered ?

Reply:
Good question. I don't think we can cover all edge cases. But when we see one we can add it to the list.